README.md

```diff
@@ -67,39 +67,39 @@ easier).
 
 ```
 $ cargo run --release --bin vtable_bench 30 10000 100000
-bench_innervtable_multialias_no_read: 1.627 +/- 0.0002
```

Comment:
So you just re-ran the experiment with the new binary names, in case that made a difference in itself?

Reply:
I did rerun them, but I didn't repost the numbers. I could do if you think that would be fairer.

Reply:
Oh, so this is just a re-ordering of the results lines?

Probably should re-run, just in case.

Reply:
Yes, because the runner sorts the benchmarks alphabetically.

The results mostly don't change *except bench_fat_with_no_read*:

```
cargo build --release && cargo run --release --bin vtable_bench 30 1000 1000000
    Finished release [optimized] target(s) in 0.02s
    Finished release [optimized] target(s) in 0.02s
     Running `target/release/vtable_bench 30 1000 1000000`
................................................................................................................................................................................................................................................
bench_fat_multialias_no_read: 1.705 +/- 0.0069
bench_fat_multialias_with_read: 1.705 +/- 0.0101
bench_fat_no_read: 1.488 +/- 0.0028
bench_fat_with_read: 2.126 +/- 0.0062
bench_innervtable_multialias_no_read: 1.641 +/- 0.0007
bench_innervtable_multialias_with_read: 1.643 +/- 0.0069
bench_innervtable_no_read: 2.126 +/- 0.0104
bench_innervtable_with_read: 2.138 +/- 0.0085
```

I'll investigate, just in case something meaningful has changed. It's possible that all we're seeing here is the jitter from a lack of bootstrapping...

Reply:
Panic over -- the mistake was mine. I had managed not to recompile some of the benchmarks after fiddling with them. Having now done so, everything has returned back to where it was in the new commit.